test/integration/targets/azure_rm_aks/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/azure
+destructive
```

Comment:
A CI group is needed here to run the integration tests as part of the pull request. Add the following line:

```
posix/ci/cloud/group5/azure
```